src/ThemeSupport/ThemeSupport.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace WPGatsby\ThemeSupport;
+
+use WPGatsby\Admin\Settings;
+
+/**
+ * Modifies the schema
+ */
+class ThemeSupport {
+	/**
+	 *
+	 */
+	function __construct() {
+        add_action( 'init', [ $this, 'registerGatsbyMenuLocations' ] );
+    }
+    
+    function registerGatsbyMenuLocations() {
+        $enable_gatsby_locations = Settings::get_setting('enable_gatsby_locations') === 'on';
+
+        if ( !$enable_gatsby_locations ) {
```

Comment:
I think the settings return `on` or `off` so this will never evaluate to false. 

So I believe you need to check `if ( 'on' !== $enable_gatsby_locations ) {`

Reply:
It's working actually :) If the setting doesn't return "on" it will turn it off. So if the setting doesn't exist and returns null or it's set to "off"

Reply:
It's evaluting that the setting is exactly equal to the string "on", anything else will trip the negative condition

Reply:
Ahh, just realized what you were thinking. I think you missed the `=== "on"` on line 19.

```
Settings::get_setting('enable_gatsby_locations') === 'on';
```

Reply:
I just flipped the side the "on" is on to make the logic easier to read

Reply:
<img width="2560" alt="Screen Shot 2020-10-16 at 12 37 05 PM" src="https://user-images.githubusercontent.com/14190743/96301717-6d735a80-0fac-11eb-909a-baf0cfc02b87.png">
<img width="2560" alt="Screen Shot 2020-10-16 at 12 36 55 PM" src="https://user-images.githubusercontent.com/14190743/96301726-7401d200-0fac-11eb-9541-8cdf63014600.png">

Reply:
ah shoot. ya, I'm used to yoda conditionals. WordPress VIP enforced that at the newspaper so ya, I did misread that line 🤦‍♂️